authentication/signUpPage/rootsignup.css

```diff
@@ -0,0 +1,41 @@
+:root {
+    --lt-cream: #f0e0c4;
```

Comment:
I would suggest to name the color little bit more explnatory, like I think by `--lt` your mean is `light`. But it is really hard to understand this, just by seeing.

Instead of that, use full name `--light-cream`.

Reply:
ok i'll change it in my future projects